.github/workflows/main.yml

```diff
@@ -15,19 +15,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        build: [linux-release, windows-release]
+        build: [linux-release, windows-release, osx-release]
         include:
           - build: linux-release
             os: ubuntu-latest
             config: release
+          - build: osx-release  
+            os: macos-14 # https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/
+            config: release            
           - build: windows-release
             os: windows-2019
             config: release
     steps:
     - uses: actions/checkout@v4
     - uses: actions/setup-dotnet@v4
       with:
-        dotnet-version: | 
+        dotnet-version: |
+          6.0.x
```

Comment:
Do we want to re-introduce 6.0? Not against it, since it's still supported until november of this year, just checking.

Reply:
This was a test before I see the debug logs. This could be removed.